source/system/Text.ooc

```diff
@@ -172,6 +172,11 @@ Text: cover {
 		this print()
 		t"\n" print()
 	}
+	read: static func (length := 64) -> This {
+		input := CString new(length)
+		fgets(input, length, stdin)
+		This new(input, length + 1)
```

Comment:
you could also pass `stdin` as parameter of type `FStream` and get a general file read function :)